### PR TITLE
Prefer constant over function

### PR DESCRIPTION
**Pull request by tesuji**
Just that I prefer constants over functions that can be made const.

### CODE REVIEW

**Review by dtolnay — APPROVED** [review]

**Comment by Manishearth** [comment]
@bors r-

https://github.com/rust-lang/rust/pull/74647#issuecomment-662646173

**Review by dtolnay — APPROVED** [review]

File: src/libstd/sys/unix/fd.rs [hunk 1]
```diff
@@ -7,24 +7,25 @@ use crate::sync::atomic::{AtomicBool, Ordering};
...
+// greater than `SSIZE_MAX` the result is "unspecified".
+//
+// On macOS, however, apparently the 64-bit libc is either buggy or
+// intentionally showing odd behavior by rejecting any read with a size
+// larger than or equal to INT_MAX. To handle both of these the read
+// size is capped on both platforms.
+#[cfg(target_os = "macos")]
+const READ_LIMIT: usize = c_int::MAX as usize - 1;
+#[cfg(not(target_os = "macos"))]
+const READ_LIMIT: usize = libc::ssize_t::MAX as usize;
```
Comment (lzutao) [inline src/libstd/sys/unix/fd.rs:28]: I wonder can we use isize::MAX instead?

**Pull request merged**